src/op-monitor-daemon/src/test/java/ee/ria/xroad/opmonitordaemon/HealthDataMetricsUtilTest.java

```diff
@@ -143,18 +145,18 @@ public void testConvertServiceIdWithDots() {
         rec.setServiceMemberClass("foo.bar");
         rec.setServiceMemberCode("testmember");
         rec.setServiceSubsystemCode("testservice_provider");
-        rec.setServiceCode("Закрой.за.мной.дверь.я.ухожу");
+        rec.setServiceCode("????????????.????.????????.??????????.??.??????????");
 
         ServiceId serviceId = HealthDataMetricsUtil.getServiceId(rec);
         String expectedServiceId = "EE/foo.bar/testmember"
-                + "/testservice_provider/Закрой.за.мной.дверь.я.ухожу";
+                + "/testservice_provider/????????????.????.????????.??????????.??.??????????";
         assertEquals(expectedServiceId, serviceId.toShortString());
 
         String requestDurationKey = getRequestDurationName(serviceId);
         assertEquals(requestDurationKey,
                 "requestDuration(EE/foo&#46;bar/testmember"
                         + "/testservice_provider"
-                        + "/Закрой&#46;за&#46;мной&#46;дверь&#46;я&#46;ухожу)");
+                        + "/????????????&#46;????&#46;????????&#46;??????????&#46;??&#46;??????????)");
```

Comment:
Lots of characters have changed in this file, intentional?

Reply:
These changes were unintentional.